src/main/java/ch/njol/skript/effects/EffConnect.java

```diff
@@ -32,62 +13,101 @@
 import ch.njol.skript.lang.SkriptParser;
 import ch.njol.skript.util.Utils;
 import ch.njol.util.Kleenean;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Connect")
-@Description("Connects a player to another bungeecord server")
-@Examples("connect all players to \"hub\"")
-@Since("2.3")
+@Description({
+	"Connects a player to another bungeecord server. This isn't required in 1.20.5+",
```

Comment:
what's "this". The whole line needs to be rewritten to be clearer about what the syntax does now